tests/test_forms.py

```diff
@@ -0,0 +1,18 @@
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm
+
+
+class FormsTest(TestCase):
+    def test_driver_creation_form_with_valid_data(self):
+        form_data = {
+            "username": "anton_best",
+            "password1": "anton12345",
+            "password2": "anton12345",
+            "first_name": "Anton",
+            "last_name": "Best",
+            "license_number": "AA0031AO",
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+        self.assertEqual(form.cleaned_data, form_data)
```

Comment:
The `assertEqual` statement compares `form.cleaned_data` with `form_data`, but the cleaned data will not include passwords in the same format. Consider checking individual fields or excluding password fields from the comparison to avoid assertion errors.